### PR TITLE
[binding] add python binding

### PR DESCRIPTION
**Pull request by robin1001**

### CODE REVIEW

**Comment by robin1001** [comment]
@Mddct , python binding, please follow the PR.

File: runtime/binding/python/py/decoder.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    def decode(self, pcm: bytes, last: bool = True) -> str:
+        """ Decode the input data
+
+        Args:
+            pcm: wav pcm
+            last: if it is the last package of the data
+        """
+        assert isinstance(pcm, bytes)
+        finish = 1 if last else 0
+        _wenet.wenet_decode(self.d, pcm, len(pcm), finish)
```
Comment (Mddct) [inline runtime/binding/python/py/decoder.py:38]: C API asserts can cause code to crash when the data is not in PCM format.
Can we add a friendly try-catch-exception

File: runtime/binding/python/py/decode_main.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import sys
```
Comment (Mddct) [inline runtime/binding/python/py/decode_main.py:1]: Movedecoder_main.py to parend direcnionary？
It‘s looks like an example

**Review by Mddct — COMMENTED** [review]
Done

File: runtime/binding/python/py/decoder.py (hunk 1 above)
Comment (robin1001) [reply runtime/binding/python/py/decoder.py:38]: Maybe refine it in the future pr.

File: runtime/binding/python/py/decode_main.py (hunk 2 above)
Comment (robin1001) [reply runtime/binding/python/py/decode_main.py:1]: it's a temp test, will refine it in the future.

**Pull request merged**